test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -784,6 +784,11 @@ foo``u{2195}abc
         { ExecuteCommand "`$herestr=@`"`n'`"'`n`"@" } | Should -Not -Throw
     }
 
+    It "A here string closing characters can be indented with spaces" {
+        { ExecuteCommand "@`"`n   here`n   string`n   `"@"} | Should -Not -Throw
```

Comment:
Please remove the line - next test do this.

Reply:
Agree. Removed.